dove/src/tests_helper.rs

```diff
@@ -106,16 +109,23 @@ pub fn set_dependencies_local_move_stdlib(project_path: &Path) {
     .unwrap();
 }
 
+/// run dove
 pub fn execute_dove_at(args: &[&str], project_folder: &Path) -> anyhow::Result<()> {
     execute(args, project_folder.to_path_buf())
 }
-pub fn execute_dove_bin_at(args: &[&str], project_folder: &Path) -> anyhow::Result<String> {
+
+/// run bin dove
+pub fn execute_dove_bin_at(
```

Comment:
You run `COMMAND_BIN_PATH *ARGS` inside a terminal. So, please, make `bin_path` the first argument.

Reply:
> Could you make this a file constant to reuse with possible future tests?

Are you sure? That was your wish